aws/backoff_retry_lambda/data_lookups.tf

```diff
@@ -0,0 +1,24 @@
+data "aws_vpc" "covid_alert" {
+  id = "vpc-06fbc7337b116cce1"
```

Comment:
Could we change this to lookup by name?  Raw ID queries make me nervous:
```terraform
data "aws_vpc" "covid_alert" {
  filter {
    name   = "tag:Name"
    values = ["CovidShield"]
  }
}
```

Reply:
We can 100% do this, I did not realize it was possible.